lib/active_triples/rdf_source.rb

```diff
@@ -106,6 +106,7 @@ def initialize(*args, &block)
       Array.wrap(self.class.type).each do |type|
         unless self.get_values(:type).include?(type)
           self.get_values(:type) << type
+          persistence_strategy.not_loaded if persistence_strategy.kind_of? ParentStrategy
```

Comment:
Is this actually required?  I don't get any test failures if I remove it.  Similarly, I wonder if it should be further up the method like so:
```ruby
      unless args.first.is_a?(Hash) || args.empty?
        set_persistence_strategy(ParentStrategy)
        persistence_strategy.parent = args.shift
        persistence_strategy.not_loaded
      else
        set_persistence_strategy(RepositoryStrategy)
      end
```